features/geometrisch-zoeken.feature

```diff
@@ -0,0 +1,327 @@
+# language: nl
+
+Functionaliteit: Als gebruiker wil ik met een geometrie (punt, bounding box) kunnen zoeken naar resources, 
+                 zodat ik gericht vervolgstappen kan definiëren m.b.t. de gevonden resources.
+
+    De resources waarvan de geometrie tenminste één punt gemeenschappelijk hebben met de opgegeven geometrie worden geretourneerd.
+
+    Bij het bepalen of een resource geometrie binnen de opgegeven contour ligt (bbox of polygoon) of 
+    dat een opgegeven geometrie (punt) binnen de resource geometrie ligt, wordt een intersects spatial operator gebruikt.
+
+    Intersects:
+    Om te bepalen of een geometrie A binnen een geometrie B ligt, deze raakt, kruist of (deels) overlapt, wordt gebruik gemaakt van de spatial operator INTERSECTS:
```

Comment:
in de specificaties hebben we geen operator (wat is dat? een parameter? een header? ...) intersects. Dus ik vraag e af wat een gebruiker met deze informatie moet